.github/CODEOWNERS

```diff
@@ -67,4 +67,146 @@ fault_tolerant_*.h                         @DataDog/debugger-dotnet
 docker-compose.serverless.yml                                      @DataDog/tracing-dotnet  @DataDog/serverless-apm
 
 # Shared code we could move to the root folder
-/tracer/build/                            @DataDog/apm-dotnet
\ No newline at end of file
+/tracer/build/                            @DataDog/apm-dotnet
+
+# IDM (note @DataDog/tracing-dotnet may be removed at a later date)
+
+## DBM and DSM
+/tracer/src/Datadog.Trace/DatabaseMonitoring/                                                           @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet
+/tracer/src/Datadog.Trace/DataStreamsMonitoring/                                                        @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet
+
+## auto instrumentations
+/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/                                       @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet
```

Comment:
I think this would be simpler, and easier to override if you set 

```
/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ @DataDog/apm-idm-dotnet @DataDog/tracing-dotnet
```

And then used overrides for the ASM and testing ones? Otherwise when you add a new instrumentation, you're not going to own it by default...

As for other teams...

- ASM should probably own/co-own `AspNet`, `AspNetCore`, `CryptographyAlgorithm`, `HashAlgorithm`, `Process`, `StackTraceLeak`
- CI vis owns `Testing`
- API owns `ManualInstrumentation`, `TraceAnnotations`, and `OpenTelemetry`
- (?)Serverless owns `Azure/Functions` and `AWS/Lambda`?

Reply:
> And then used overrides for the ASM and testing ones? Otherwise when you add a new instrumentation, you're not going to own it by default...

I didn't know you could do that 
But that sounds a lot more maintainable, so I'll swap to do that and override the ones for the teams you listed. Thanks!

> you'll get pinged whenever ASM/CI vis work on an integration 😬

Yeah that's only for now - soon it'll be for multiple languages :)